npm/private/npm_package_store.bzl

```diff
@@ -238,6 +238,13 @@ def _npm_package_store_impl(ctx):
                         "--directory",
                         package_store_directory.path,
                     ],
+                    # no need to cache tarball extraction for npm package remotely. Untar is faster than uploading/downloading
+                    # cache in most of the cases, and untar creates mutliple outputs and sometimes the number of the output can be massive
+                    # meaning it can takes much more time to upload the cache compared to the actual untar.
+                    # see https://github.com/aspect-build/rules_js/pull/1974
+                    execution_requirements = {
```

Comment:
I'd believe this is a speedup for local builds, but I think it's a pessimization for remote builds. Same exact reasoning as the execution requirements on bazel-lib `CopyFile` applies here. So let's make this configurable if we do land it

Reply:
I remembered the original CopyFile/Directory execution requirements didn't play well with BWOB and was worried about the same thing here.

Reply:
Right, the problem is:
- if inputs are local, and subsequent actions that depend on the outputs will run local, you want this action to run local and skip writing the result to remote cache
- if inputs are remote, or subsequent actions are remote, you want this action to run remote and store result in the remote cache

So there isn't a reasonable default we can use for everyone. Perhaps a better answer is to document that you can use `--modify_execution_info= NpmPackageExtract=+no-remote` if you find your builds are better tuned with that setting

Reply:
another possibility is to add that flag to a preset, like https://github.com/bazel-contrib/bazel-lib/blob/main/.aspect/bazelrc/javascript.bazelrc - this way more devs will discover it